lib/util/failover_uri.js

```diff
@@ -168,6 +168,9 @@ shaka.util.FailoverUri.prototype.createRequest_ =
         // effectively cache every response.
         this.requestPromise_ = null;
         this.request_ = null;
+	if(xhr.responseURL) {
```

Comment:
I'm fixing with a different pull request, which includes the information you needed and has had lint.sh run